Include/internal/pycore_optimizer.h

```diff
@@ -208,6 +208,9 @@ struct _Py_UOpsAbstractFrame {
     _Py_UopsSymbol **stack_pointer;
     _Py_UopsSymbol **stack;
     _Py_UopsSymbol **locals;
+
+    // Might be NULL if unable to statically determine.
```

Comment:
Allowing a NULL pointer makes this harder to use.
I think it would be better if the symbol were `sym_new_unknown` when it isn't known.

Reply:
OK I used `sym_new_not_null`